doc/api/api_changes.rst

```diff
@@ -11,6 +11,19 @@ sources of the changes you are experiencing.
 For new features that were added to matplotlib, please see
 :ref:`whats-new`.
 
+Changes in 2.1.0
+================
+
+Code changes
```

Comment:
@anntzer That sounds like a good plan to me.

Wish we were 3 only and could use enums :disappointed: